test/packages/good/manifest.yml

```diff
@@ -40,3 +40,8 @@ icons:
     title: system
     size: 1000x1000
     type: image/svg+xml
+build:
```

Comment:
There are pros and cons for both options.

`it will also stay in the package when downloaded by Kibana. Do we want that?`

Pros: 
* Kibana can link directly to ECS docs or repo to refer to the revisions
* We don't need to reverse engineer the ECS version used by the built package

Cons:
* Manifest contains new entries for dependencies

`Or should it be really a build time definition that is removed?`

I would be against modifying the manifest to get rid of dependencies. Maybe we can get rid of the `build` level and just leave dependencies.

`Maybe a separate file?`

Hm.. we can extract this logic to `deps.yml`. WDYT?

Reply:
> One advantage of depds.yml is that there could be a command elastic-package deps update (just made it up) that automatically updates references in the deps.yml file. I would not necessarily do the same with the manifest.

Yes, that's the plan. I was thinking about `elastic-package build --update`, which will try to pull latest commit references.